crypto/s2n_ecc_evp.c

```diff
@@ -478,21 +480,25 @@ int s2n_ecc_evp_parse_params_point(struct s2n_blob *point_blob, struct s2n_ecc_e
     return 0;
 }
 
-int s2n_ecc_evp_parse_params(struct s2n_ecdhe_raw_server_params *raw_server_ecc_params,
-                             struct s2n_ecc_evp_params *ecc_evp_params) {
+int s2n_ecc_evp_parse_params(struct s2n_connection* conn,
+                                struct s2n_ecdhe_raw_server_params* raw_server_ecc_params,
+                                struct s2n_ecc_evp_params* ecc_evp_params) {
```

Comment:
Nit: If you're going to touch this code, can you make the wrapping more normal instead of even stranger? :)